tracking/src/main/java/org/hps/recon/tracking/kalman/SvtHitEfficiencyKalman.java

```diff
@@ -0,0 +1,1831 @@
+    Map<String, IHistogram2D> AllHitsnumberOfTracksTanLVsPhi = new HashMap<String, IHistogram2D>();
+    Map<String, IHistogram2D> AllHitsnumberOfTracksWithHitOnMissingLayerTanLVsPhi = new HashMap<String, IHistogram2D>();
+
+    Map<String, IHistogram1D> AllHitsnumberOfTracksChannelEle = new HashMap<String, IHistogram1D>();
+    Map<String, IHistogram1D> AllHitsnumberOfTracksWithHitOnMissingLayerChannelEle = new HashMap<String, IHistogram1D>();
+    Map<String, IHistogram1D> AllHitsnumberOfTracksUEle = new HashMap<String, IHistogram1D>();
+    Map<String, IHistogram1D> AllHitsnumberOfTracksWithHitOnMissingLayerUEle = new HashMap<String, IHistogram1D>();
+    Map<String, IHistogram1D> AllHitsnumberOfTracksPEle = new HashMap<String, IHistogram1D>();
+    Map<String, IHistogram1D> AllHitsnumberOfTracksWithHitOnMissingLayerPEle = new HashMap<String, IHistogram1D>();
+    Map<String, IHistogram1D> AllHitsnumberOfTracksTanLEle = new HashMap<String, IHistogram1D>();
+    Map<String, IHistogram1D> AllHitsnumberOfTracksWithHitOnMissingLayerTanLEle = new HashMap<String, IHistogram1D>();
+    Map<String, IHistogram1D> AllHitsnumberOfTracksPhiEle = new HashMap<String, IHistogram1D>();
+    Map<String, IHistogram1D> AllHitsnumberOfTracksWithHitOnMissingLayerPhiEle = new HashMap<String, IHistogram1D>();
+    Map<String, IHistogram2D> AllHitsnumberOfTracksTanLVsPhiEle = new HashMap<String, IHistogram2D>();
+    Map<String, IHistogram2D> AllHitsnumberOfTracksWithHitOnMissingLayerTanLVsPhiEle = new HashMap<String, IHistogram2D>();
+
+    Map<String, IHistogram1D> AllHitsnumberOfTracksChannelPos = new HashMap<String, IHistogram1D>();
+    Map<String, IHistogram1D> AllHitsnumberOfTracksWithHitOnMissingLayerChannelPos = new HashMap<String, IHistogram1D>();
+    Map<String, IHistogram1D> AllHitsnumberOfTracksUPos = new HashMap<String, IHistogram1D>();
+    Map<String, IHistogram1D> AllHitsnumberOfTracksWithHitOnMissingLayerUPos = new HashMap<String, IHistogram1D>();
+    Map<String, IHistogram1D> AllHitsnumberOfTracksPPos = new HashMap<String, IHistogram1D>();
+    Map<String, IHistogram1D> AllHitsnumberOfTracksWithHitOnMissingLayerPPos = new HashMap<String, IHistogram1D>();
+    Map<String, IHistogram1D> AllHitsnumberOfTracksTanLPos = new HashMap<String, IHistogram1D>();
+    Map<String, IHistogram1D> AllHitsnumberOfTracksWithHitOnMissingLayerTanLPos = new HashMap<String, IHistogram1D>();
+    Map<String, IHistogram1D> AllHitsnumberOfTracksPhiPos = new HashMap<String, IHistogram1D>();
+    Map<String, IHistogram1D> AllHitsnumberOfTracksWithHitOnMissingLayerPhiPos = new HashMap<String, IHistogram1D>();
+    Map<String, IHistogram2D> AllHitsnumberOfTracksTanLVsPhiPos = new HashMap<String, IHistogram2D>();
+    Map<String, IHistogram2D> AllHitsnumberOfTracksWithHitOnMissingLayerTanLVsPhiPos = new HashMap<String, IHistogram2D>();
+
+
+    /*
+    Map<String, IHistogram1D> numberOfTracksChannelCorrected = new HashMap<String, IHistogram1D>();
```

Comment:
is this section going to be relevant at some time or will it stay as a comment forever?